Import the Zed ADRV9002 dtsi renames onto 2019_R2 to match with ADI Linux branch 2019_R2

m4eme opened the pull request:
Renamed ADRV9002 Zed pl-delete-nodes dtsi files to match changes made to ADI Linux Kernel 2019_R2 branch.

nunojsa reviewed (commented):
Thanks for this. I'm currently on vacations so I really appreciate this PR to push a fix for this. However I would like you to improve your commit message to something like (run git log to check examples on the history messages):

```
meta-adi-xilinx: devicetree: rename adrv9002 zynq files

"Then here explain that the files changed in the kernel tree and we need to change it here..."
```

On top of the improved message, we need a Signed-of tag. Run `git commit -s` when comitting your change.

m4eme commented:
Thanks Nuno.
I came across this in a project and since I had to do the rename anyway I thought I could just submit a PR for it.
Changed the commit message and added the signed tag. Enjoy your vacation.

nunojsa reviewed (approved):
Yes, I always appreciate that someone else contributes here.

Thanks for the fix!
